Foundation/NSURLSession/http/EasyHandle.swift

```diff
@@ -454,7 +453,7 @@ fileprivate extension _EasyHandle {
             }
             var length = Double()
             try! CFURLSession_easy_getinfo_double(handle.rawHandle, CFURLSessionInfoCONTENT_LENGTH_DOWNLOAD, &length).asError()
```

Comment:
I guess Ubuntu 16.04  comes with a [default libcurl 7.47](https://launchpad.net/ubuntu/xenial/+source/curl). I am not sure if we should be using an API that is newer than 7.47.